engine/flink/kafka-util/src/test/scala/pl/touk/nussknacker/engine/kafka/source/flink/KafkaSourceFactoryProcessMixin.scala

```diff
@@ -54,7 +60,7 @@ trait KafkaSourceFactoryProcessMixin extends FunSuite with Matchers with KafkaSo
   }
 
   after {
```

Comment:
I wonder, maybe we can put it in FlinkSpec?

Reply:
https://github.com/TouK/nussknacker/pull/2484/commits/dd2818b975a391c5ddc42f2335c8be2b316042fe